homework2/script.js

```diff
@@ -0,0 +1,72 @@
+const menuIcon = document.querySelector(".menu-icon");
+menuIcon.addEventListener("click", function () {
+    const navigate = document.getElementsByTagName("nav");
```

Comment:
bad idea to get element by tag name, use class name or even id if your element is unique

Reply:
Done